repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts

```diff
@@ -29,8 +29,41 @@ export const getUniquePriceGroups = (fareZones: FareZone[]): string[] => [
 
 export const getIdName = (name: string): string => name.replace(/(\s)+/g, '_');
 
-export const getPointToPointScheduledStopPointsList = (fareZones: FareZone[]): ScheduledStopPoints[] =>
-    getStops(fareZones).map(stop => ({
+export const existsTwiceInArray = (array: string[], element: string): boolean => {
```

Comment:
Should we get rid of this as the 'hasDuplicates' function is the one we use?

Reply:
Good catch, I forgot to remove that.